CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing
+
+This file provides general guidance for developers that are creating or updating REST API definitions for Azure.
+
+## Table of Contents
+<!--
+  The TOC below is generated using the `markdown-toc` node package.
+
+      https://github.com/jonschlinkert/markdown-toc
+
+  You should regenerate the TOC after making changes to this file.
+
+      markdown-toc -i --maxdepth 4 CONTRIBUTING.md
```

Comment:
You could just abbreviate this to have users run:
```bash
npx markdown-toc -i --maxdepth 4 CONTRIBUTING.md
```
Anyone that knows `npx` will now that you can run `npm info markdown-toc` to see info. `npx` takes care of pulling the package and effectively running `npm run` on it.

Just a thought.